composer/_version.py

```diff
@@ -0,0 +1,6 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""The Composer Version."""
+
+__version__ = '0.8.0'
```

Comment:
I think by default we should append it with 'dev' or something similar so that users know if they are on dev or a released version.

Reply:
Discussed offline; gonna leave this out cause python doesn't respect semantic versioning.